pom.xml

```diff
@@ -102,10 +104,7 @@
 							<goal>sign</goal>
 						</goals>
 						<configuration>
-							<gpgArguments>
-								<arg>--pinentry-mode</arg>
-								<arg>loopback</arg>
-							</gpgArguments>
+						    <defaultKeyring>false</defaultKeyring>
```

Comment:
I think we want to add back in these lines as these just provided extra configuration to the GPG command-line tool, which I believe are still needed:
```
        <gpgArguments>
            <arg>--pinentry-mode</arg>
            <arg>loopback</arg>
        </gpgArguments>
```

Reply:
Additionally, when disabling the default key ring, you need to provide the location of a custom key ring with it, like this:
```
<defaultKeyring>false</defaultKeyring>
    <keyring>${project.basedir}/path/to/custom/keyring.gpg</keyring>
```

Although can you please explain why we would not use the default keyring here?